src/Data/Textile/Inputs.elm

```diff
@@ -139,7 +137,6 @@ type alias Query =
     , fading : Maybe Bool
     , dyeingMedium : Maybe DyeingMedium
     , printing : Maybe Printing
-    , ennoblingHeatSource : Maybe HeatSource
```

Comment:
This is a breaking change for the API, so we must add an entry to the Api changelog. Also, we should update the `openapi.yml` docs accordingly.

Reply:
Sure, thx for noticing